code-challenges.test.js

```diff
@@ -17,38 +20,93 @@
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array of that length containing the numbers of the Fibonacci.
 
 // a) Create a test with expect statements for each of the variables provided.
+test('when nums is called, you should get an array containing the fibonacci numbers for number that you input', () => {
+    let fibNumbers = [1, 1, 2, 3, 5, 8]
+    expect(nums).toEqual(fibNumbers)
+})
 
 // Example input: 6
+let nums1= 6
+// b) Create the function that makes the test pass.
+// declare a function that will store the array
+const nums = [1, 1];
+// // created a for loop that went through the array and extended and added numbers as long as the length was less than 10
+for(var i=nums.length; i<nums1; i++) {
+//     // add numbers starting from the next position then adding it to the previous
+    nums[i] = nums[i-2] + nums[i-1];
+}
+console.log(nums); 
 // Expected output: [1, 1, 2, 3, 5, 8]
 
-// Example input: 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
-
-
-
-
+test('when nums is called, you should get an array containing the fibonacci numbers for number that you input', () => {
+    let fibNumbers = [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+    expect(numsArray).toEqual(fibNumbers)
+})
 
+// Example input: 10
+  let nums2= 10
 // b) Create the function that makes the test pass.
-
-
+// declare a function that will store the array
+const numsArray = [1, 1];
+// // created a for loop that went through the array and extended and added numbers as long as the length was less than 10
+for(var i=numsArray.length; i<nums2; i++) {
+//     // add numbers starting from the next position then adding it to the previous
+    numsArray[i] = numsArray[i-2] + numsArray[i-1];
+}
+console.log(numsArray); 
+// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 
 
 // --------------------2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+
+test('when newArray is called, you should get a new array with only odd numbers and they should be sorted from least to greatest as a return', () => {
+    let fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+    let sortedArray = newArray(fullArr1)
+    expect(sortedArray).toEqual([-9, 7, 9, 199])
+})
+
 var fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
+const newArray = (array) => {
```

Comment:
```suggestion
const getSortedOddValues = (array) => {
```
Try to name functions so that they describe the logic of the function